airflow_provider_fivetran/hooks/fivetran.py

```diff
@@ -0,0 +1,242 @@
+from time import sleep
+from urllib.parse import urlparse
+
+import requests
+from requests import PreparedRequest, exceptions as requests_exceptions
+from requests.auth import AuthBase
+import pendulum
+import logging
+import json
+import time
+
+
+from airflow import __version__
+from airflow.exceptions import AirflowException
+from airflow.hooks.base_hook import BaseHook
+
+log = logging.getLogger(__name__)
+
+class FivetranHook(BaseHook):  # noqa
```

Comment:
I have no idea what `# noqa` does, but we likely don't need it here. Probably a Databricks code coverage thing.